gum/backend-darwin/gumprocess-darwin.c

```diff
@@ -532,22 +615,44 @@ gum_module_enumerate_ranges (const gchar * module_name,
     if (lc->cmd == GUM_LC_SEGMENT)
     {
       gum_segment_command_t * segcmd = (gum_segment_command_t *) lc;
+      gboolean is_page_zero;
       GumPageProtection cur_prot;
 
+      is_page_zero = segcmd->vmaddr == 0 &&
+          segcmd->filesize == 0 &&
+          segcmd->vmsize != 0 &&
+          (segcmd->initprot & VM_PROT_ALL) == VM_PROT_NONE &&
+          (segcmd->maxprot & VM_PROT_ALL) == VM_PROT_NONE;
+
```

Comment:
Let's drop this blank line so these two look like part of the same paragraph/higher level operation.